newrelic-agent/src/main/java/com/newrelic/agent/instrumentation/weaver/ClassWeaverService.java

```diff
@@ -300,9 +280,7 @@ public void registerInstrumentationCloseable(String instrumentationName, Closeab
     /**
      * Load all the weave packages embedded in the agent jar.
      */
-    private Collection<ClassMatchVisitorFactory> loadInternalWeavePackages() {
-        final Collection<ClassMatchVisitorFactory> matchers = Sets.newConcurrentHashSet();
-
+    private void loadInternalWeavePackages() {
```

Comment:
This is the original method that got copied for the security agent. Same changes from there.